src/open_inwoner/accounts/tests/test_profile_views.py

```diff
@@ -193,6 +193,58 @@ def test_save_filled_form(self):
         self.assertEquals(self.user.postcode, "1013 RM")
         self.assertEquals(self.user.city, "Amsterdam")
 
+    def test_save_with_invalid_first_name_chars_fails(self):
+        invalid_characters = "/\"\\,.:;'"
+
+        for char in invalid_characters:
+            with self.subTest(char=char):
+                response = self.app.get(self.url, user=self.user)
+                self.assertEquals(response.status_code, 200)
```

Comment:
This is pattern that happens a lot in this codebase, but you don't need the separate `self.assertEquals(response.status_code, 200)` because the WebTest client already tests for the status code. Although the default accepts both some 2xx and 3xx, if if it must be just 200 you need to specify it:

`response = self.app.get(self.url, user=self.user, status=200)`